packages/plugin-utils/src/resolveOptions.ts

```diff
@@ -226,7 +227,8 @@ export async function loadConfiguration(options: LoadConfigurationOptions) {
           // hack to prevent `import` get transformed
           // eslint-disable-next-line no-new-func
           const _import = new Function('modulePath', 'return import(modulePath)')
-          resolved = (await _import(configFilePath))?.default || {}
+
+          resolved = (await _import(pathToFileURL(resolve(configFilePath))))?.default || {}
```

Comment:
not quite sure if we can get this working without `resolve()`
to improve it further.. with resolve it works

Reply:
It should already be resolved, have you tried without it?

Reply:
I do check that tomorrow again, to make sure... if we get full path like `C://` I think we can drop `resolve`